app-tasks/rf/src/rf/ingest/source.py

```diff
@@ -0,0 +1,53 @@
+""" Python class to represent the sources within an ingest layer """
+class Source():
+    def __init__(self, image, extent, starting_target_band, band_maps=None,
+                 crs="epsg:3857", extent_crs="epsg:3857"):
+
+        """
+            Create a new ingest Source
+
+            Args:
+                image (rf.models.Image): Image instance the source is based on
+                extent (List[float]): extent of the source
+                starting_target_band (int): index of first target band
+                band_maps (List[dict]): list of mappings for each band within the image
+                crs (str): CRS of the Source (@TODO: figure out why this is needed)
+                extent_crs (str): CRS of the extent (@TODO: figure out why this is needed)
```

Comment:
Actually - this might be a question for @moradology or @echeipesh. 

I think you are correct though -- the CRS of the extent is almost always going to be `epsg:4326`